javascript/javascript1/week1/index.js

```diff
@@ -23,3 +23,43 @@ if (shouldShowResultInDogYears) {
 else {
     console.log('Your dog will be ' + dogYear + ' human years old in ' + dogYearFuture);
 }
+
+// Housey pricey (A house price estimator)
+
+
+// Peter Hous
+
+const petersWide = 8;
+const petersDeep = 10;
+const petersHigh = 10;
+const peterGardenSizeInM2 = 100,
+
+const volumeInMeters = petersWide * petersDeep * petersHigh;
+const housePrice = volumeInMeters * 2.5 * 1000 + peterGardenSizeInM2 * 300;
+
+const peterHousCost = 2.500.000;
+
+if (housePrice < peterHousCost)
+    console.log('Peter will pay too little');
+else (housePrice > peterHousCost)
+console.log('Peter will pay too much');
```

Comment:
This code is actually not working as it should. This happens because you are adding a condition to the `else` statement, which is not supposed to be there since else does not take any conditions and only runs if the `if` statement condition is not met. Also, you are not using curly braces in the statements `{}`, which is not totally wrong, but it's also not recommended, since it can get confusing or even lead to errors if you want to add more `if` or `else` blocks in the future. to avoid bad practice, try using the curly braces.
You can read more about if-elsestatement here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else